openassessment/xblock/test/test_staff_area.py

```diff
@@ -781,3 +781,50 @@ def _create_submission(item, values, types):
         peer_api.on_start(submission["uuid"])
         workflow_api.create_workflow(submission["uuid"], types)
         return submission
+
+    @scenario('data/self_only_scenario.xml', user_id='Bob')
+    def test_staff_area_student_info_image_submission_names(self, xblock):
+        """
+        This test checks if file name is rendered on Manage-individual-learner template ie oa_student_info.html.
+
+        Context: In table submission_submissions raw_answer column in db is long text and used as json field ,
```

Comment:
Better but still needs improvement. We don't have to give back-end/data persistency information in the docstring. They need to explain what is being tested. In our case, we are testing that the file name will be rendered for both `file_names` and `files_names`. Make this test a ddt test and check it for both these keys.

Reply:
Update test and added ddt for both keys